health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/ViewAttendance.js

```
@@ -2,104 +1,566 @@
+          disabledAction ? (
+            <Button
+              label={t(`HCM_AM_GO_BACK`)}
+              title={t(`HCM_AM_GO_BACK`)}
+              onClick={() => {
+                fromCampaignSupervisor ? history.push(`/${window.contextPath}/employee/payments/generate-bill`, { fromViewScreen: true }) :
+                  history.push(`/${window.contextPath}/employee/payments/registers-inbox`);
+              }}
+              type="button"
+              style={{ minWidth: "14rem" }}
+              variation="primary"
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)